elements/rh-audio-player/rh-audio-player.css

```diff
@@ -444,11 +443,30 @@ rh-menu > button:focus {
   text-align: right;
 }
 
-#play,
-#full-play {
-  border-radius: 50%;
-  background-color: var(--_surface-color);
-  color: var(--rh-audio-player-icon-background-color, var(--_text-color));
+#full-play::part(button) {
+  width: var(--_button-size);
+  height: var(--_button-size);
+}
+
+#play::part(button) {
+  --rh-icon-size: var(--rh-size-icon-01, 16px);
+
+  width: var(--_button-size);
+  height: var(--_button-size);
+}
+
+#play::part(button),
+#full-play::part(button) {
+  /* danger! overwriting an rh-button internal var, do what I say not as I do. */
+  --_play-background-color: var(--rh-color-surface-darkest, #151515);
+}
+
+.dark {
+  #play::part(button),
+  #full-play::part(button) {
+    /* danger! overwriting an rh-button internal var, do what I say not as I do. */
+    --_play-background-color: var(--rh-color-surface-lightest, #ffffff);
+  }
```

Comment:
I don't like doing this, but I must override an internal CSS variable in the rh-button to match the Figma styles. @bennypowers, as you suggested, we can break this out into a separate issue. Maybe go with the broken styles while still using the `rh-button variant="play"` in this PR. @marionnegp, what are your thoughts?

Reply:
After a discussion with @marionnegp we are going to back out of the usage of `rh-button variant=play` for now and reimplement the styles needed to adjust the icon that is already in rh-button so we can release this PR still as a patch. There was another issue with this approach, and that is that the `rh-button variant=play` doesn't support swapping its icon to a pause button easily, even if we can just use the part to override the background colors necessary for audio-player.